packages/cms-cli/commands/customObject/batch.js

```diff
@@ -0,0 +1,9 @@
+const createCommand = require('./batch/create');
```

Comment:
Instead of having a batch command what if we have a single command that expects the JSON file to be an array of objects? The thinking is that we should strive for a minimal command surface area when we can.

Reply:
Sure we can do that.  Are you saying also combine the `batch/create` and `create`, or just bring `batch/create` up a level to be something like `create-multiple`?

Reply:
Combine into a single `hs custom-object create` command. The thinking is that the most common use case is creating multiple so a little friction for creating a single object is not the end of the world. It also mirrors how I image other situations might work like `hs blog-post create` or `hs page create` — both of which have issues (e.g. #38 and #181).